lesson15_task1.py

```diff
@@ -0,0 +1,24 @@
+"""
+A Person class
+Make a class called Person. Make the __init__() method take firstname, lastname, and age as parameters
+and add them as attributes. Make another method called talk() which makes prints a greeting from the person containing,
+for example like this:
+"Hello, my name is Carl Johnson and I’m 26 years old"
+"""
+
+
+class Person:
```

Comment:
Very nice!
Only thing you could refactor:
instead of checking error: `self.age = int(age)`
you could use:
`if not isinstance(age, int): 
    raise ValueError`

BTW, for such mandatory type specifications Pydantic library exists, it could be also a useful thing for you to learn.